services/entity-server/src/middleware/requests.js

```diff
@@ -44,14 +44,22 @@ export const requestSessionData = dispatch => {
 
 export const requestProjectListData = dispatch => {
   axios
-    .get(backendServiceBaseUrl, sparqlProjectListParams())
+    .get(backendServiceBaseUrl, sparqlProjectListRequest())
     .then(result => {
       frameData(result.data, "gi2mo:IdeaContest")
-      .then(data => {
-        dispatch(extractProjectList(data))
-      })
+        .then(data => {
+          dispatch(extractProjectList(data))
+        })
     })
```

Comment:
does this really need to run async ? if yes, then good, if no , no need for .then()

Reply:
Async is enforced by the JSON-LD library (by digitalbazaar). I'm really not sure why they do this, but for me it felt like the request.js is a good point to deal with it